src/model/import.ts

```diff
@@ -59,6 +59,9 @@ export class ScannedImport implements Resolvable {
   }
 
   resolve(document: Document): Import|undefined {
+    if (!document.analyzer.canResolveUrl(this.url)) {
```

Comment:
I think this is the right thing for now, as @usergenic says, it only remove the warning. Next step would be to preserve the feature in a way that the bundler ignores, and possibly add warnings on the _imports_ for the ones that deserve it.